src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/taskconfiguration/services/TaskAutoAssignmentService.java

```diff
@@ -119,14 +125,22 @@ public boolean checkAssigneeIsStillValid(TaskResource taskResource, String assig
         List<RoleAssignment> orderedRoleAssignments = orderRoleAssignments(rolesList, roleAssignments);
         return orderedRoleAssignments.stream()
             .filter(roleAssignment -> {
```

Comment:
For extra safety we could also add add an extra  to check for roleAssignment.getRoleName() not empty
```

```

Reply:
I checked this scenario and added a test for it, this method orderRoleAssignments is filtering out any role assignment with a null or empty rolename